### DISCUSSION TITLE
Automation on Users activity (login, password change etc.)

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3409] [Help] [answered]
Hi,
I am trying to understand if there is a way to get user login activity. And I suppose I can use automation to create a log or alert. But could not figure out how to setup the trigger. 
Is there any guide on this? Thanks!

**Comment by mjashanks** [comment, marked as answer]
This is not possible right now, we do not log these events. When we do start to do this, it will be covered by a wider auditing feature.

Also asked here: https://github.com/Budibase/budibase/discussions/2149

**Reply by micebrain** [reply]
ok, thanks
